app/src/main/java/com/mediaan/mymediaan/view/ftu/CreateProfileScreen.kt

```diff
@@ -176,13 +190,41 @@ fun CreateProfileScreen(
                 maxValue = ftuViewModel.maxCharFacts,
                 labelId = R.string.create_profile_screen_lie_label
             )
+            Text(
+                text = stringResource(id = R.string.create_profile_screen_heading_interests),
+                modifier = Modifier
+                    .padding(top = 24.dp)
+                    .padding(bottom = 16.dp),
+                style = Typography.bodyLarge,
+                fontWeight = FontWeight.SemiBold,
+            )
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(480.dp)
+            ) {
+                items(Interest.entries.toTypedArray()) { interest ->
+                    Row(
+                        verticalAlignment = androidx.compose.ui.Alignment.CenterVertically,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .clickable { toggleInterest(interest) }
+                    ) {
+                        Checkbox(
+                            checked = selectedInterests.contains(interest),
+                            onCheckedChange = { toggleInterest(interest) },
+                            colors = CheckboxDefaults.colors(checkedColor = MediaanPrimary)
+                        )
```

Comment:
When the whole row is clickable, I think we can replace Checkbox with simple icon and pass it empty or checked depending on interest state. Or do you see why we would benefit from using native Checkbox in this case?